### PR TITLE
feat(template): enhanced the PR template to match Jira instead of the old github issues

### PR DESCRIPTION
**Pull request by RSickenberg**
### 💬 Describe the pull request/code changes
_A clear and concise description of what the pull request is about._

### 🎟️ Jira
_This pull request is related to the following Jira issue(s):_

### 🔢 To Review
_Steps to review the PR:_
1. 👷 🏗 🚧

### 📸 Screenshots
_If applicable, add screenshots to help explain the task._

### 📝 Checklist
- [ ] Update the "Unreleased" section of the `CHANGELOG.md` with [chan](https://github.com/geut/chan/tree/main/packages/chan).  _(if applicable)_
- [ ] The PR is covered by some tests. _(if applicable)_

[comment]: <> (Add the Jira ticket if needed/applicable)
[comment]: <> (Add steps to review the PR if needed/applicable)
[comment]: <> (Add others conditions to allow the PR to be merged if needed/applicable)

---
#### Emojis
1. 👷 🏗 🚧
2. 🤔 💭
3. 🧑🏻‍💻 _Need more coffee_
4. 🫣
#### Text
1. _Hey, it's a construction zone! Come back later..._
2. _WIP_
3. _`Working on it..`_
#### Hack
1.
2.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-### 🔖 Does it need an entry in the CHANGELOG ?
+### 🔖 Does it need an entry in the CHANGELOG?
  - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`
 
 ### 📸 Screenshots
-If applicable, add screenshots to help explain the task.
+_If applicable, add screenshots to help explain the task._
+
+### 📦 Ready to merge
+- [ ] The reviewer approves the pull request.
```
Comment (gido) [inline .github/pull_request_template.md]: We already have this information with GitHub. What is the reason to duplicate it?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+- [ ] The PR is covered by some tests. _(if applicable)_
+- [ ] Merges conflicts are resolved. _(if applicable)_
```
Comment (gido) [inline .github/pull_request_template.md]: Same here, GitHub UI already give the information that there is a conflict

**Comment by gido** [comment]
I'm happy that `#1` and `#2` disappear 🫠!

File: .github/pull_request_template.md (hunk 2 above)
Comment (RSickenberg) [reply .github/pull_request_template.md]: Make sens, can remove

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 
 ### 📸 Screenshots
-If applicable, add screenshots to help explain the task.
+_If applicable, add screenshots to help explain the task._
+
+### 📦 Ready to merge
+- [ ] The PR is covered by some tests. _(if applicable)_
+
+[comment]: <> (Add steps to review the PR if needed/applicable)
+[comment]: <> (Add others conditiosn to allow the PR to be merged if needed/applicable)
```
Comment (gido) [inline .github/pull_request_template.md]:
```suggestion
[comment]: <> (Add others conditions to allow the PR to be merged if needed/applicable)
```

**Review by gido — APPROVED** [review]

**Comment by WengerK** [comment]
I think we should keep a checkbox for "ready to merge" and "has a changelog entry" nop ? why having free-text section for those 2 ?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,20 +1,19 @@
...
 
 ### 📸 Screenshots
-If applicable, add screenshots to help explain the task.
+_If applicable, add screenshots to help explain the task._
+
+### 📝 Checklist
+- [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`. _(if applicable)_
+- [ ] The PR is covered by some tests. _(if applicable)_
+
+[comment]: <> (Add the Jira issue)
```
Comment (RSickenberg) [inline .github/pull_request_template.md]:
```suggestion
[comment]: <> (Add the Jira ticket if needed/applicable)
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -1,20 +1,19 @@
...
-### 🔖 Does it need an entry in the CHANGELOG ?
- - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`
+_Steps to review the PR:_
 
 ### 📸 Screenshots
-If applicable, add screenshots to help explain the task.
+_If applicable, add screenshots to help explain the task._
+
+### 📝 Checklist
+- [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`. _(if applicable)_
```
Comment (yannlugrin) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Update the "Unreleased" section of the `CHANGELOG.md`.  _(if applicable)_
```

To match our current requirements

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -1,20 +1,19 @@
...
-- #2
+### 🎟️ Jira
+_This pull request is related to the following Jira issue(s):_
 
 ### 🔢 To Review
-Steps to review the PR:
-1. Install '...'
-2. Go to '...'
-3. Click on '....'
-4. Scroll down to '....'
```
Comment (Yago) [inline .github/pull_request_template.md:14]: Keep that or at least something similar; steps are too often skipped or unexplained.

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -1,20 +1,19 @@
...
-### 🔖 Does it need an entry in the CHANGELOG ?
- - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`
+_Steps to review the PR:_
 
 ### 📸 Screenshots
-If applicable, add screenshots to help explain the task.
+_If applicable, add screenshots to help explain the task._
+
+### 📝 Checklist
+- [ ] Update the "Unreleased" section of the `CHANGELOG.md`.  _(if applicable)_
```
Comment (Yago) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Update the "Unreleased" section of the `CHANGELOG.md` with [chan](https://github.com/geut/chan/tree/main/packages/chan).  _(if applicable)_
```

**Review by Yago — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 6 above)
Comment (RSickenberg) [reply .github/pull_request_template.md:14]: I agree, but I often delete all the steps to rewrite it. It's asked in the comments on the last lines

Comment (yannlugrin) [reply .github/pull_request_template.md:14]: I agree that we could let that, even if we delete it, this example is a better incentive to write something,

Comment (RSickenberg) [reply .github/pull_request_template.md:14]: So @yannlugrin and @Yago can we agree to find maybe a better example or at least, less points ?

Comment (yannlugrin) [reply .github/pull_request_template.md:14]: Write a proposal :)

Comment (RSickenberg) [reply .github/pull_request_template.md:14]: @yannlugrin Check the proposals on the readme of the PR.

Comment (WengerK) [reply .github/pull_request_template.md:14]: I think having 2-3 multipoints on the "step to review" is more interesting, with 1 point, people will tend to add only 1 sentence (my point of view)

Comment (RSickenberg) [reply .github/pull_request_template.md:14]: @WengerK Personally, I still have to delete all the lines to rewrite them afterward bc placeholders like those don't fit my needs at all. Maybe the 1. 2. 3. empty is the best solution so in my case I could reuse it.

Comment (Gurakh) [reply .github/pull_request_template.md:14]: I agree with with them, 2-3 multipoints is a better incentive to write clear review steps. Plus it's really not that tedious to remove if needed.

Comment (RSickenberg) [reply .github/pull_request_template.md:14]: Ok, but can we agree to have something more fluent or appealing? Usually, I have to remove all the text from the steps. 

Have a look in the body of the PR for some ideas.

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -1,20 +1,20 @@
...
-Steps to review the PR:
-1. Install '...'
-2. Go to '...'
-3. Click on '....'
-4. Scroll down to '....'
-
-### 🔖 Does it need an entry in the CHANGELOG ?
- - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`
+_Steps to review the PR:_
+1. 👷 🏗 🚧
```
Comment (gido) [inline .github/pull_request_template.md]:
```suggestion
1. 
2. 
```

**Review by yannlugrin — APPROVED** [review]

**Review by gido — APPROVED** [review]

**Comment by gido** [comment]
Merci a tous!

**Comment by WengerK** [comment]
<3

**Pull request merged**